modules/tensor_mechanics/doc/content/source/materials/ADLAROMANCEStressUpdateBase.md

```diff
@@ -1,10 +1,9 @@
-# AD LAROM Creep Stress Update
-
-!syntax description /Materials/ADLAROMCreepStressUpdate
+# LAROMANCE Stress Update with Automatic Differentiation
 
 ## Description
 
-`ADLAROMCreepStressUpdate` computes the creep rate of materials by sampling a Reduced Order Model (ROM) formulated
+The `ADLAROMANCEStressUpdateBase` class computes the creep rate of materials by sampling a Los Alamos Reduced Order
+Model Applied to Nonlinear Constitutive Equations (LAROMANCE) formulated
 via calibration with lower-length scale simulations. This method utilizes the exact same techniques
```

Comment:
The sentence starting in this line could potentially be misleading....at first I thought the 'This method' was talking about the calibration, not the implemented class. New paragraph maybe?

Reply:
just added `ADLAROMANCEStressUpdateBase` to make it explicit